slider/collection.py

```diff
@@ -0,0 +1,98 @@
+# TODO: not use replay private functions
+from .replay import _consume_int, _consume_string
```

Comment:
I think we could just move these helpers to something like `utils` or `parsing_utils` and make them public.

Reply:
I plan on moving them to utils.py

Reply:
do you want to do that in a future change, or should we do that before merging this?

Reply:
I moved `consume_*` (not tied to replay) 
https://github.com/jxu/slider/commit/8a89826c205dff67f0a372f70d5efd051e977b25